datasets/hess-crab4/make_tables.py

```diff
@@ -11,13 +11,16 @@
 rows = []
 for obs in obs_ids:
     for filetype in file_types:
-        name = "hess_" + filetype + "_" + obs + ".fits.gz"
+        #name = "hess_" + filetype + "_" + obs + ".fits.gz"
+        name = "hess_{}_{:06d}.fits.gz".format(filetype, obs)
         if filetype == 'events':
-            name = "hess_events_simulated_" + obs + ".fits"
+        #    name = "hess_events_simulated_" + obs + ".fits"
+            name = "hess_events_simulated_{:06d}.fits".format(obs)
         if filetype == 'background':
-            name = "hess_bkg_offruns_" + obs + ".fits.gz"
+        #    name = "hess_bkg_offruns_" + obs + ".fits.gz"
+            name = "hess_bkg_offruns_{:06d}.fits.gz".format(obs)
         data = dict()
-        data['OBS_ID'] = int(obs[1:])
```

Comment:
I am getting this error:
"""
TypeError: 'int' object has no attribute '__getitem__'
"""